api/errors/builder.go

```diff
@@ -0,0 +1,93 @@
+package errors
+
+// This file contains all of the errors that can be generated from the
+// docker/builder component.
+
+import (
+	"net/http"
+
+	"github.com/docker/distribution/registry/api/errcode"
+)
+
+var (
+	// ErrorCodeAtLeastOneArg is generated when the parser comes across a
+	// Dockerfile command that doesn't have any args.
+	ErrorCodeAtLeastOneArg = errcode.Register(errGroup, errcode.ErrorDescriptor{
```

Comment:
Mapping of actions from error code doesn't seem in scope for this as we can't possibly know what actions people should take - that's a UI, or CLI type of consideration.  All we can do is present the caller with the unique error for each condition - they can then decide what to do.  If we allow two different conditions to share the same error then there's ambiguity and that's not good.